observability/distr_tracing/distr-tracing-rn.adoc

```diff
@@ -12,6 +12,171 @@ You can use the {TempoName} xref:../otel/otel-forwarding-telemetry-data.adoc#ote
+==== Deprecated functionality
+
+* In the {TempoName} 3.4, ???.
+* In the {TempoName} 3.4, ???.
+* In the {TempoName} 3.4, ???.
+
+[id="distr-tracing_3-4_tempo-release-notes_removal-notice_{context}"]
+==== Removal notice
+
+In the {TempoName} 3.4, the FEATURE has been removed. Bug fixes and support are provided only through the end of the 3.? lifecycle. As an alternative to the FEATURE for USE CASE, you can use the ALTERNATIVE instead.
+
+[id="distr-tracing_3-4_tempo-release-notes_bug-fixes_{context}"]
+==== Bug fixes
+
+This update introduces the following bug fixes:
+
+* Before this update, ... . With this update, this issue is fixed by ... . (link:https://issues.redhat.com/browse/TRACING-????[TRACING-????])
+* Before this update, ... . With this update, this issue is fixed by ... . (link:https://issues.redhat.com/browse/TRACING-????[TRACING-????])
+* Before this update, ... . With this update, this issue is fixed by ... . (link:https://issues.redhat.com/browse/TRACING-????[TRACING-????])
+* Before this update, ... . With this update, this issue is fixed by ... . (link:https://issues.redhat.com/browse/TRACING-????[TRACING-????])
+
+[id="distr-tracing_3-4_tempo-release-notes_known-issues_{context}"]
+==== Known issues
+
+There is currently a known issue:
+
+* Currently, the {TempoShortName} fails on the {ibm-z-title} (`s390x`) architecture. (link:https://issues.redhat.com/browse/TRACING-3545[TRACING-3545])
+* Currently, ???. (link:https://issues.redhat.com/browse/TRACING-????[TRACING-????])
+* Currently, ???. (link:https://issues.redhat.com/browse/TRACING-????[TRACING-????])
+* Currently, ???. (link:https://issues.redhat.com/browse/TRACING-????[TRACING-????])
+
+[id="distr-tracing_3-4_jaeger-release-notes_{context}"]
+=== {JaegerName}
+
+The {JaegerName} 3.4 is provided through the link:https://catalog.redhat.com/software/containers/rhosdt/jaeger-operator-bundle/613b50f3a052bb39f2c5f31e/history[{JaegerOperator} Operator <x>.<y>.<z>].
+
+[NOTE]
+====
+The {JaegerName} 3.4 is based on the open source link:https://www.jaegertracing.io/[Jaeger] release 1.<y>.0.
+====
+
+[IMPORTANT]
+====
+Jaeger does not use FIPS validated cryptographic modules.
+====
+
+[id="distr-tracing_3-4_jaeger-release-notes_support-for-elasticsearch-operator_{context}"]
+==== Support for the {es-op}
+
+The {JaegerName} 3.4 is supported for use with the {es-op} 5.6, 5.7, and 5.8.
+
+[id="distr-tracing_3-4_jaeger-release-notes_deprecated-functionality_{context}"]
+==== Deprecated functionality
+
+In the {DTProductName} 3.4, Jaeger and support for Elasticsearch remain deprecated, and both are planned to be removed in a future release.
+Red Hat will provide support for these components and fixes for CVEs and bugs with critical and higher severity during the current release lifecycle, but these components will no longer receive feature enhancements.
+
+The {JaegerOperator} Operator will be removed from the `redhat-operators` catalog in November 2025.
```

Comment:
@jgomezselles, is this sentence enough or do you still want me to include the link to this page: https://access.redhat.com/solutions/7083722?

Reply:
Thanks! If it is a common practice to link these articles, I think it won't hurt adding the link because it provides more context to users that need more context. If adding the link is an issue, I'm totally fine with the sentence. Thanks again!

Reply:
Thank you for clarifying, Jose. I've added the link to the solution page just as you requested.